.github/workflows/release.yml

```
@@ -0,0 +1,69 @@
+        cp *.h *.c Makefile LICENSE README.md CONTRIBUTING.md dist/FastCSV-C-${GITHUB_REF#refs/tags/}/
+        cp -r tests dist/FastCSV-C-${GITHUB_REF#refs/tags/}/
+        cd dist && tar -czf FastCSV-C-${GITHUB_REF#refs/tags/}.tar.gz FastCSV-C-${GITHUB_REF#refs/tags/}/
+
+    - name: Create binary distribution
+      run: |
+        mkdir -p dist/FastCSV-C-${GITHUB_REF#refs/tags/}-linux-x64
+        cp *.h libcsv.so libcsv.a LICENSE README.md dist/FastCSV-C-${GITHUB_REF#refs/tags/}-linux-x64/
+        cd dist && tar -czf FastCSV-C-${GITHUB_REF#refs/tags/}-linux-x64.tar.gz FastCSV-C-${GITHUB_REF#refs/tags/}-linux-x64/
+
+    - name: Create Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)